misc/scripts/install-local.sh

```diff
@@ -38,6 +38,7 @@ function cleanup() {
 		sudo rm -rf "${SRCDIR:?}"/*
 		sudo rm -rf /tmp/pacstall/*
 	fi
+	sudo rm -f /tmp/pacstall-func
```

Comment:
This shouldn't be using sudo

Reply:
Actually, this shouldn't be here anymore after my last refactor

Reply:
Nvm, I mistaken it with another file